.github/workflows/ruby_tests.yml

```
@@ -1,81 +1,20 @@
+---
 name: Ruby Testing
-on:
-  pull_request:
-env:
-  BUNDLE_WITHOUT: journald:development:console:libvirt
-  RAILS_ENV: test
-  DATABASE_URL: postgresql://postgres:@localhost/test
-  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
+on: pull_request
```

```suggestion
on:
  pull_request:
  push:
    branches:
      - 'master'
      - '*-stable'
```

otherwise there won't be no cache, which I find rather important to have :)

---

We should expand the actions readme with this (and multible Foreman versions)

---

Are you spying on my vim buffers?!

---

https://github.com/theforeman/actions/pull/34